### PR TITLE
Assignment completed

### PR DESCRIPTION
**Pull request by jjacob2012**
I first forked the assignment and then I created a new version control on R studio.

I began by adding my name to the README.md file, putting it as a header and then bolding it. Then, I provided a brief description of my academic interests, followed by a list of my hobbies and favorite foods. Throughout the assignment, I constantly referred to the Markdown cheatsheet that I pulled from the Help tab in R studio and played around with the aesthetic functions. I accidentally tumbled upon a box feature when I was playing around with the "create a new paragraph" function - I was trying to see if two additional spaces on top of the original two would automatically create two subsequent lines.

I fulfilled 4 of the Markdown syntax requirements though I attempted to include an image to the file. I was unable to do so because the following prompt came up: 

[WARNING] This document format requires a nonempty <title> element.
  Please specify either 'title' or 'pagetitle' in the metadata,
  e.g. by using --metadata pagetitle="..." on the command line.
  Falling back to 'README.utf8'

To which I retracted the code and stuck with the submitted format.

Towards the end, I was a bit worried because a prompt appeared in R studio indicating that my file was too large. Fearing that my data might go unsaved, I copy-pasted it to a word file for future reference. Fortunately, I was able to successfully submit the assignment. I really like the diff and commit features in R studio because it thoroughly keeps track of changes made to my document.

### CODE REVIEW

**Comment by choubd** [comment]
Great job! Things I really like about:
- Great use of the required elements;
- Clean structure with the use of lists and heads;
- Great introduction!
I think there is not much to improve, but maybe you want to try adding some images. Also, I noticed that the link did not quite work, probably because of the double "http://".

**Comment by skimbo** [comment]
Hi Jonathan, great submission! I like how its very organized and easy to follow. I also had trouble with embedding images, and I had comments on my post as to how to correctly do them so take a look if you'd like. Also, while not completely necessary, you could hyperlink the youtube link in the form of: [PewDiePie](youtube link here). Sidenote: I'm also interested in your research concentration, would love to see what you can produce from this class towards your topic.

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01 
+# **Jonathan Jacob**
```
Comment (nearridge) [inline README.md:2]: FYI placing this item in a header will automatically bold it so the bolding isn't necessary here.

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+I am a MAPSS student concentrating in education. My research centers around the role of sports in education. Specifically, the ways in which sports participation can manifest itself as a navigational tool for social mobility and stability, *particularly* among low-income students.
```
Comment (nearridge) [inline README.md:3]: The emphasis on "particularly" is effective here however.

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01 
+# **Jonathan Jacob**
+I am a MAPSS student concentrating in education. My research centers around the role of sports in education. Specifically, the ways in which sports participation can manifest itself as a navigational tool for social mobility and stability, *particularly* among low-income students. 
+  
+## **Hobbies:**
+  
+  - playing video games
```
Comment (nearridge) [inline README.md:7]: Great job with this list.

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |   x           |            |
| **Reproducibility**         |       x    |              |            |

Remarks: Nice job on this! No major problems I found. Consider trying things out with images or other files in future assignments!

**Pull request closed without merging**